.tekton/push.yaml

```
@@ -41,3 +41,22 @@ spec:
     - name: git-auth
       secret:
         secretName: '{{ git_auth_secret }}'
+
+  finally:
+    - name: notify-failure
+      when:
+        - input: $(tasks.status)
+          operator: in
+          values: ["Failed", "Completed"]
+      params:
+        - name: message
+          value: |
+            :x: Pipeline `<$(context.pipelineRun.name)>` has failed or completed with issues.
+            See details: <https://console.redhat.com/application-pipeline/workspaces/cost-mgmt-dev/applications/nise-populator/pipelineruns/$(context.pipelineRun.name)|Pipeline Run>.
+        - name: secret-name
+          value: "slack-webhook-notification-secret"
+        - name: key-name
+          value: "webhook-url"
+      taskRef:
+        name: slack-webhook-notification
+        bundle: quay.io/konflux-ci/tekton-catalog/task-slack-webhook-notification:0.1@sha256:dc17b70633363d78414b8c06dc1660d25742935f106a6116995638e1210c2730
```

I think this taskRef might be incorrect
```suggestion
        resolver: bundles
        params:
          - name: name
              value: slack-webhook-notification
          - name: bundle
            value: quay.io/konflux-ci/tekton-catalog/task-slack-webhook-notification:0.1@sha256:dc17b70633363d78414b8c06dc1660d25742935f106a6116995638e1210c2730
            - name: kind
               value: task
```